lib/checks/forms/fieldset.js

```diff
@@ -50,28 +50,62 @@ function checkARIAGroup(group, name) {
 	return true;
 }
 
+function checkTable (group, nodes) {
```

Comment:
Yeah those checkboxes don't qualify as a group. Only ones that have the same name attribute. If they do have the same name attribute they need a group label. That is what this rule is all about. So we should only pass checkbox groups that actually have some form of label. The question is how do we identify that within a table?